### DISCUSSION TITLE
Pivot table name for roles and permissions

### DISCUSSION THREAD

**Discussion by mrkazmierczakuw** [spatie/laravel-permission#2167] [Ideas]
Probably the name of pivot table form Spatie Permissions package should be change to permission_role according to Laravel pivot table rules. How do you think?

**Comment by erikn69** [comment]
You can change everything as you want
https://github.com/spatie/laravel-permission/blob/dbc6c18ef4b2d097350a5faf82382564748e6c97/config/permission.php#L71

**Reply by mrkazmierczakuw** [reply]
Yes, I know this, but according to the pivot table rules singular form and alphabetical order it should be change in the package.

**Reply by erikn69** [reply]
That would be a **breaking change** for everyone, will you give support to everybody on that case?

**Reply by mrkazmierczakuw** [reply]
Backward compatibility is possible. This can be an option or core change.

**Reply by erikn69** [reply]
Fell free to make a PR
